packages/yargs/lib/webpack.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
```

Comment:
Thoughts of a freshman: providing unit tests for such a module would extremely help any future maintainer understanding what's going on here. :)

Reply:
That is probably true - given the integrative, I/O-heavy nature of most of `untool`, I have postponed the addition of unit tests to a later point. E.g., to comprehensively unit test this plugin thing, we would not just have to mock a logger, but also a major part of webpack's plugin API.